middleware/src/main/java/br/ufrn/dimap/middleware/remotting/impl/ClientRequestHandlerImpl.java

```diff
@@ -0,0 +1,98 @@
+package br.ufrn.dimap.middleware.remotting.impl;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+
+import br.ufrn.dimap.middleware.remotting.interfaces.ClientProtocolPlugin;
+import br.ufrn.dimap.middleware.remotting.interfaces.ClientRequestHandler;
+
+/**
+ * The Client Request Handle is responsible for sending data to the server.
+ * Follows Singleton pattern.
+ * 
+ * @author victoragnez
+ */
+public final class ClientRequestHandlerImpl implements ClientRequestHandler {
```

Comment:
Remove the "Impl" at the end, since interface and class are in different packages and it is clear that it is an implementation.

Reply:
In order to avoid using the qualified name, we will keep the "Impl" suffix in this case.